docs/configure-gcp.md

```diff
@@ -0,0 +1,117 @@
+# Configuring Google Cloud Plattform GCP
+
+The provisioning service uses Google cloud compute engine to perform all provisioning tasks.
+We try to follow the best possible security standard and practices.
+In this article, we will describe how to create necessary service account setup and configure this application with it.
+
+## Configuring provisioning role
+
+There are two accounts used in the provisioning flow:
+
+* **Service account** which is configured using GOOGLE_APPLICATION_CREDENTIALS environment variable which holds a credentials JSON file.
+    this is an account of the service provider (e.g. Red Hat for cloud.redhat.com).
+    It needs a Role with a set of permissions that enable the application to perform operations. 
+    These are list machine types, import pubkey, create new instance, start and stop.
+
+* **Tenant account** is the account into which the service account switches via GCP's IAM permissions to API operation.
+    It needs to specify the permissions it gives to the service account when adding it as an IAM.
+
+You will need to know Email of the Service account.
+For development and testing purposes, you can use the same account for both.
+
+### Service account
+
+This is the account that this application needs the credentials and will use to connect to GCP services.
+
+Create two projects from two different google accounts, one served as the service account's project and one as the tenant's project. 
+
+#### Service account Configuration
+
+Create a new Service account:
+
+1. Navigate to the Service account project
+2. Navigate to IAM & Admin on GCP.
+3. Click Service Accounts
+4. Click Create Service Account.
+5. Fill in the Service Account details.
+6. Click CREATE AND CONTINUE.
+7. Choose the permissions the provisioning app needs (TBC):
+
+Roles:
+ - Service Account User Role
+  
+Permissions:
+  - compute.instances.insert
```

Comment:
Permissions need to be added when we will add a listing of machine types and importing pubkeys